Corona-Warn-App/build.gradle

```diff
@@ -166,6 +166,12 @@ android {
                 srcDirs = ['src/device']
             }
         }
+        test {
+            java.srcDirs += "$projectDir/src/testShared/java"
+        }
+        androidTest {
+            java.srcDirs += "$projectDir/src/testShared/java"
```

Comment:
same folder?

Reply:
Yes, shared folder for test classes used in both unit and instrumentation tests, needs to be added to `test` and `androidTest` to be available.